src/network/changesManager/changesmanager.cpp

```diff
@@ -0,0 +1,75 @@
+#include "changesmanager.h"
+
+ChangesManager::ChangesManager()
+{
+
+}
+
+bool ChangesManager::getChange(const QString & newFile, Change & change, const SizeType fileId)
+{
+    QString currentFile = m_currentFiles.at(static_cast<int>(fileId));
```

Comment:
m_currentFiles.
Using our convection, we don't use _ for variables names

Reply:
This is outdated file and is absent in context of build